spec/rubocop/cop/style/variable_number_spec.rb

```diff
@@ -97,6 +97,50 @@
       expect(cop.offenses.size).to eq(0)
     end
 
+    it 'does not register an offense for normal case number in local
+      variable' do
+      inspect_source(cop, 'user1_id = 1')
+      expect(cop.offenses.size).to eq(0)
+    end
+
+    it 'does not register an offense for normal case number in local
+      variable' do
+      inspect_source(cop, 'sha256 = 3')
+      expect(cop.offenses.size).to eq(0)
+    end
+
+    it 'does not register on offense for normal case multi digit number in
+      local variable' do
+      inspect_source(cop, 'foo10_bar = 4')
+      expect(cop.offenses.size).to eq(0)
+    end
+
+    it 'does not register on offense for normal case multi digit number in
+      local variable' do
+      inspect_source(cop, 'foo_bar10 = 4')
+      expect(cop.offenses.size).to eq(0)
+    end
+
+    it 'does not register an offense for normal case number in the middle of
+      local variable' do
+      inspect_source(cop, 'target_u2f_device = nil')
+      expect(cop.offenses.size).to eq(0)
+    end
+
+    it 'registers an offense for only integers in the middle' do
+      inspect_source(cop, 'user_1_id = 3')
+      expect(cop.offenses.size).to eq(1)
+      expect(cop.highlights).to eq(['user_1_id'])
+      expect(cop.messages).to eq(['Use normalcase for variable numbers.'])
```

Comment:
I thought it was a good idea to write tests generally the same way as others tested similar functionality. It seemed to me like a fail safe to ensure that the offenses are registered for the right reason so I decided to incorporate those checks into my tests as well. I can remove them if you find the check unnecessary, though.